src/CodeGen_WebAssembly.cpp

```diff
@@ -136,6 +136,11 @@ string CodeGen_WebAssembly::mattrs() const {
         sep = ",";
     }
 
+    if (target.has_feature(Target::WasmThreads)) {
+        s << sep << ",+atomics,+bulk-memory";
```

Comment:
atomics I get, but why do we need bulk-memory here? (I'm sure there's a good reason, just can't remember)

Reply:
I was not able to figure that out. There's an error message telling you to do it. It makes me worry there's some mass copying of memory going on.